FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Mauro Worobiej on 05/09/2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+class FeedImageMapper {
+	private struct Root: Decodable {
+		let items: [ItemModel]
+
+		var feedImage: [FeedImage] {
+			return items.map { $0.feedImage }
+		}
+	}
+
+	private struct ItemModel: Decodable {
+		let id: UUID
+		let description: String?
+		let location: String?
+		let url: URL
+
+		enum CodingKeys: String, CodingKey {
+			case id = "image_id"
+			case description = "image_desc"
+			case location = "image_loc"
+			case url = "image_url"
+		}
+
+		var feedImage: FeedImage {
+			FeedImage(id: id, description: description, location: location, url: url)
+		}
+	}
+
+	static var OK_200: Int { 200 }
```

Comment:
`OK_200` is an implementation detail of the `FeedImageMapper` so this property should be marked as `private`.
```suggestion
private static var OK_200: Int { 200 }
```

Reply:
got it, thanks!